spec/ParseLiveQuery.spec.js

```diff
@@ -310,14 +312,10 @@ describe('ParseLiveQuery', function () {
     Parse.Cloud.afterLiveQueryEvent('TestObject', req => {
       const current = req.object;
       const original = req.original;
-
-      setTimeout(() => {
-        done();
-      }, 2000);
-
       if (current.get('foo') != original.get('foo')) {
         req.sendEvent = false;
       }
+      setTimeout(done, 1000);
```

Comment:
unclear why this is 1000msec, reducing will probably not cause the test to fail but might falsely have it succeed..